src/diffusers/models/attention.py

```diff
@@ -447,16 +446,6 @@ def __init__(
         # 3. Feed-forward
         self.norm3 = nn.LayerNorm(dim)
 
-        # if xformers is installed try to use memory_efficient_attention by default
```

Comment:
We should not automatically enable xformers for two reasons:

- 1. It's not PyTorch-like API. PyTorch doesn't by default enable all the fastest options available
- 2. We allocate GPU memory before the user even does `.to("cuda")` 
- 3. This behavior is not consistent with cases where xformers is not installed

Reply:
cc @pcuenca @anton-l @patil-suraj

Reply:
Related issues: https://github.com/huggingface/diffusers/issues/1565
https://github.com/huggingface/diffusers/issues/1344#issuecomment-1335017683